### PR TITLE
Remove mapbox references from docs and CLI tools

### LINKED ISSUE #111: Remove mapbox references in docs
There are some places that still link to the `mapbox://` protocol and use some example that has the `mapbox` name in them.
All of these should be replaced with mapblire and http(s):// protocol.

### PR DESCRIPTION
**Pull request by typebrook**
@HarelM 
This PR resolves #111:

- Test: Replace source urls with assets from demotiles.maplibre.org
- Test: Remove tests for Mapbox API
- Doc: Replace urls with `mapbox://` protocol
- Feature: Remove migration step for Mapbox API
- Update CHANGELOG

## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

**Comment by typebrook** [comment]
All patterns about mapbox protocol (`mapbox://`) are removed, use GNU grep for check:

```shell
# No pattern found besides CHANGELOG and git folder
grep -R 'mapbox://' | grep -Ev '(^CHANGELOG.md|^.git)'
```

File: test/integration/style-spec/tests/many-sources.input.json [hunk 1]
```diff
@@ -1,14 +0,0 @@
-{
-  "version": 8,
-  "sources": {
-    "one": {
-      "type": "vector",
-      "url": "mapbox://mapbox.mapbox-streets-v6"
```
Comment (HarelM) [inline test/integration/style-spec/tests/many-sources.input.json:6]: Can't we simply change the address instead of removing the test?

**Comment by HarelM** [comment]
Can you check for the word mapbox in general, not just the protocol?

File: test/integration/style-spec/tests/many-sources.input.json (hunk 1 above)
Comment (typebrook) [reply test/integration/style-spec/tests/many-sources.input.json:6]: As I mentioned in commit message, this test case is only used for check limitation about Mapbox API (Max 15 sources), see https://docs.mapbox.com/api/maps/vector-tiles/#retrieve-vector-tiles

**Comment by typebrook** [comment]
> Can you check for the word mapbox in general, not just the protocol?

Still has media assets from https://static-assets.mapbox.com, should we keep them?
```shell
# Get "mapbox" pattern in URL route
> grep -R '/mapbox' | grep -Ev '(^CHANGELOG.md|^package.json|^package-lock.json|^node_modules|github.com/mapbox/)'
docs/src/routes/sources/index.tsx:        "https://static-assets.mapbox.com/mapbox-gl-js/drone.mp4",
docs/src/routes/sources/index.tsx:        "https://static-assets.mapbox.com/mapbox-gl-js/drone.webm"
```

**Comment by HarelM** [comment]
Are they used or just fake? If they are used then it's OK I guess, or we can bring static assets to this repo...

**Comment by typebrook** [comment]
The others reference to blog post or docs pages which doesn't included in maplibre.org. So we should keep them.

```shell
# Get "mapbox.com" pattern
> grep -R 'mapbox.com' | grep -Ev '(^CHANGELOG.md|^package.json|^package-lock.json|^node_modules|github.com/mapbox/)' | grep mapbox.com
docs/src/routes/sources/index.tsx:A vector tile source. Tiles must be in [Mapbox Vector Tile format](https://docs.mapbox.com/vector-tiles/). All geometric coordinates in vector tiles must be between \`-1 * extent\` and \`(extent * 2) - 1\` inclusive. All layers that use a vector source must specify a [\`"source-layer"\`](${import.meta.env.BASE_URL}layers/#source-layer) value.
docs/src/routes/sources/index.tsx:A raster DEM source. Only supports [Mapbox Terrain RGB](https://blog.mapbox.com/global-elevation-data-6689f1d0ba65) and Mapzen Terrarium tiles.
docs/src/routes/sources/index.tsx:        "https://static-assets.mapbox.com/mapbox-gl-js/drone.mp4",
docs/src/routes/sources/index.tsx:        "https://static-assets.mapbox.com/mapbox-gl-js/drone.webm"
docs/src/routes/layers/index.tsx:The [Vintage map style](https://blog.mapbox.com/designing-the-vintage-style-in-mapbox-studio-9da4aa2a627f) uses a custom SVG [\`background-pattern\`](${import.meta.env.BASE_URL}layers/#paint-background-background-pattern) to achieve a textured vintage look.
docs/src/routes/layers/index.tsx:This map of a [Strava](https://blog.mapbox.com/strava-launches-gorgeous-new-outdoor-maps-977c74cf37f9) user's hike through Grand Teton National Park uses the [\`line-color\`](${import.meta.env.BASE_URL}layers/#paint-line-line-color) and [\`line-width\`](${import.meta.env.BASE_URL}layers/#paint-line-line-width) paint properties to style the strong red line of the user's route.
docs/src/routes/layers/index.tsx:This map uses an external dataset to provide data-driven values for the [\`fill-extrusion-height\`](${import.meta.env.BASE_URL}layers/#paint-fill-extrusion-fill-extrusion-height) paint property of various [country polygons](https://blog.mapbox.com/high-resolution-administrative-country-polygons-in-studio-57cf4abb0768) in a fill-extrusion layer.
src/reference/v8.json:          "doc": "Mapbox Terrain RGB tiles. See https://www.mapbox.com/help/access-elevation-data/#mapbox-terrain-rgb for more info."

**Comment by typebrook** [comment]
> Are they used or just fake? If they are used then it's OK I guess

They are used in example in maplibre.org: https://maplibre.org/maplibre-gl-js-docs/example/video-on-a-map/

> , or we can bring static assets to this repo...

Let me check the copyright!

**Comment by HarelM** [comment]
We should generate a small video and a webp image I guess, it shouldn't be hard.

**Comment by HarelM** [comment]
If these are the only references left I think we are good to go.

**Review by HarelM — APPROVED** [review]

**Comment by typebrook** [comment]
Thanks for quick review.

> We should generate a small video and a webp image I guess, it shouldn't be hard.

Would you mind search bird view video from opendata (Like wiki commons), or maplibre can send someone outdoor (with drone or air-balloon I guess?) to record a video?

**Pull request merged**